### PR TITLE
Update w info from docs

### PR DESCRIPTION
**Pull request by crandmck**
@0candy 
Per https://github.com/strongloop/loopback.io/issues/155 I am copying info from the docs into the README.  Then we'll pull in the README into the docs so we only have to maintain the info in one place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,363 @@
 # loopback-connector-mssql
 
-`loopback-connector-mssql` is the Microsoft SQL Server connector module for [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler/).
+[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/default.aspx) is a relational database management system developed by Microsoft.
+The `loopback-connector-mssql` module is the Microsoft SQL Server connector for the LoopBack framework.
 
-For complete documentation, see [Documentation | SQL Server Connector](http://loopback.io/doc/en/lb2/SQL-Server-connector.html).
+<div class="gh-only">
+For more information, see [LoopBack documentation | SQL Server Connector](http://loopback.io/doc/en/lb2/SQL-Server-connector.html).
```
Comment (0candy) [inline README.md]: LB3 link: http://loopback.io/doc/en/lb3/SQL-Server-connector.html

File: README.md [hunk 2]
```diff
@@ -1,47 +1,363 @@
...
+    "url": "mssql://username:password@dbhost/dbname",
+    "database": "mydb",
+    "password": "admin",
+    "user": "admin",
+  }
+```
+
+Edit `datasources.json` to add other properties that enable you to connect the data source to a SQL Server database.
+
+To connect to a SQL Server instance running in Azure, you must specify a qualified user name with hostname, and add to the following to the data source declaration:
```
Comment (0candy) [inline README.md]: and add ~~to~~ the following to the data source declaration:

File: README.md [hunk 3]
```diff
@@ -1,47 +1,363 @@
...
+
+### Auto-migratiion
+
+The SQL Server connector also supports _auto-migration_ that enables you to create a database schema
+from LoopBack models using the [LoopBack automigrate method](http://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-automigrate).
+For each model, the LoopBack SQL Server connector creates a table in the 'dbo' schema in the database.
+
+For more information on auto-migration, see [Creating a database schema from models](https://loopback.io/doc/en/lb3/Creating-a-database-schema-from-models.html) for more information.
+
+Destroying models may result in errors due to foreign key integrity. First delete any related models first calling delete on models with relationships.
```
Comment (0candy) [inline README.md]: First delete any related models ~~first~~ `by` calling delete on models with relationships.

**Comment by crandmck** [comment]
Good catches, thanks again!!

**Pull request merged**